ui/desktop/app/templates/scopes/scope/projects/targets.hbs

```diff
@@ -71,12 +74,6 @@
                         <:key>{{t 'form.type.label'}}</:key>
                         <:value>{{credential.library.type}}</:value>
                       </dropdown.key-value>
-                      {{#if credential.library.name}}
```

Comment:
I think it's worth keeping this in the dropdown for visibility.  It's another way to get the same information, particularly because in the card header the key isn't labeled.